source/camera/basic/capture.py

```diff
@@ -8,9 +8,10 @@ def _main():
     camera = app.connect_camera()
 
     with camera.update_settings() as updater:
-        updater.settings.iris = 40
-        updater.settings.exposure_time = datetime.timedelta(microseconds=40000)
-        updater.settings.filters.reflection.enabled = True
+        updater.settings.iris = 11.31
```

Comment:
The iris is different here compared to the cpp sample. Typo?

https://github.com/zivid/zivid-cpp-samples/blob/master/source/Camera/Basic/Capture/Capture.cpp

Reply:
The iris values are available in the porting guide on the KB.